script/Arcueid/Day 7/07_00_ARC07_1.txt

```diff
@@ -247,12 +247,12 @@ I hear the faint rustling of clothes and the sound of approaching footsteps.
 [sha:1fb1ab4f2cfc0d85849edbffd8deae451b4229a9]{
 -- Page 18, Offset 16611.
 -- 「あ……いや、熱は、確かにあるん、だけど」
-"Ah... no, it's true that I have a fever, but..."
+"Ah... no, it's true I have a fever, but..."
 }
 [sha:d55c297ab6106e9cbdaa128dc3bcbc76893bf740]{
 -- Page 19, Offset 16612.
 -- 秋葉は遠慮なく近づいてきて、下から俺の顔を覗きこむ。
-Akiha walks right up next to me and peers up into my face from below.
+Akiha walked right up next to me and peered up into my face from below.
```

Comment:
she walks even before her voice line

Reply:
Man tenses in VNs is a philosophy on its own, should I change tenses in the future edits in similar cases?

Reply:
yeah it definitely sucks, especially since at least nasu has a tendency to flop back and forth. I don't think we can apply a hard and fast rule, but the src text here is present.